docs/01-programs/02-candy-machine/04-inserting-items.md

```diff
@@ -2,15 +2,200 @@
+
+</div>
+</AccordionItem>
+</Accordion>
+
+## Overriding Existing Items
+
+When inserting items, you may provide the position in which these items should be inserted. This enables you to insert items in any order you want but also allows you to update items that have already been inserted.
+
+Note that, once minting has started, items can no longer be changed.
+
+<Accordion>
+<AccordionItem title="JS SDK" open={true}>
+<div className="accordion-item-padding">
+
+The following examples show how you can insert 3 items and, later on, update the second item inserted.
+
+```tsx
+await metaplex.candyMachines().insertItems({
+  candyMachine,
+  items: [
+    { name: "My NFT #1", uri: "https://example.com/nft1.json" },
+    { name: "My NFT #2", uri: "https://example.com/nft2.json" },
+    { name: "My NFT #3", uri: "https://example.com/nft3.json" },
+  ],
+});
+
+await metaplex.candyMachines().insertItems({
+  candyMachine,
+  index: 1,
+  items: [{ name: "My NFT #X", uri: "https://example.com/nftX.json" }],
+});
+
+candyMachine = await metaplex.candyMachines().refresh(candyMachine);
+candyMachine.items[0].name; // "My NFT #1"
+candyMachine.items[1].name; // "My NFT #X"
+candyMachine.items[2].name; // "My NFT #3"
+```
+
+API References: [Operation](https://metaplex-foundation.github.io/js/classes/js.CandyMachineClient.html#insertItems), [Input](https://metaplex-foundation.github.io/js/types/js.InsertCandyMachineItemsInput.html), [Output](https://metaplex-foundation.github.io/js/types/js.InsertCandyMachineItemsOutput.html), [Transaction Builder](https://metaplex-foundation.github.io/js/classes/js.CandyMachineBuildersClient.html#insertItems).
 
-This page is not ready yet but we’re working hard on documenting it. Check back a bit later.
+</div>
+</AccordionItem>
+</Accordion>
 
-In the meantime, you can learn more about Candy Machine V3 via the following resources:
+## Conclusion
 
-- [Candy Machine Core Program documentation](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/candy-machine-core/program)
-- [Candy Guard Program documentation](https://github.com/metaplex-foundation/mpl-candy-guard)
+And just like that, we have a loaded Candy Machine ready to mint NFTs! However, we've not created any requirements for our minting process. How can we configure the price of the mint? How can we ensure that buyers are holders of a specific token or an NFT from a specific collection? How can we set the start date of our mint? What about the end conditions?
 
-Thank you!
+[On the next page](/programs/candy-machine/candy-guards), we’ll talk about Candy Guards which make all of this possible.
```

Comment:
We should add a note to say that minting is only allowed when all config lines have been added – we added a specific check on CMv3 for that.

Reply:
Good shout!